### DISCUSSION TITLE
[백준] 2206 벽 부수고 이동하기

### DISCUSSION THREAD

**Discussion by pykido** [BDD-CLUB/challenge-every-week-posting#453] [4. 기타 글 모음]
- 이게 맞는데 왜 틀렸지?를 수십 번 외친 문제입니다 ㅠㅠㅠ... 
- 포기할 뻔하다가 반례를 통해 문제점을 찾았습니다 ! 
- 더욱 정진하겠습니당

[블로그 바로가기 ](https://animoto1.tistory.com/entry/%EB%B0%B1%EC%A4%80-2206-%EB%B2%BD-%EB%B6%80%EC%88%98%EA%B3%A0-%EC%9D%B4%EB%8F%99%ED%95%98%EA%B8%B0-C)

**Comment by jojaeng2** [comment]
이 문제는 어려운 문제가 맞습니다.. 저도 옛날에 이 문제에서 머리가 많이 빠졌던 기억이 나네요 ㅎ

**Reply by pykido** [reply]
저도 알고리즘 풀었을 때의 카타르시스와 알고리즘 푸는 과정에서 빠지는 머리카락을 자주 등가교환하곤 합니다 ㅎ
